qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -1253,32 +1265,32 @@ class Kubernetes implements OrchestratorMain {
         def namespaces = evaluateWithRetry(2, 3) {
             return client.namespaces().list().items.find {
                 it.getMetadata().getName() == name
-            }.collect {
+            }.collect { Namespace ns ->
```

Comment:
This was a problem for a compiler as we are already in a closure so we will override `it` with different type so I renamed it to `ns`